src/state/HeartwoodContextProvider.js

```diff
@@ -27,28 +27,31 @@ const getCookie = (name) => {
 }
 
 const deleteSpaces = (input) => {
-  
-  let a = input.replace(/[\u200B-\u200D\uFEFF]/g, 'invisiblecharacter')
-  
-  let b = a.split('invisiblecharacter')
+  let a = input.replace(/[\u200B-\u200D\uFEFF]/g, "invisiblecharacter")
+
+  let b = a.split("invisiblecharacter")
 
   let c = b.join("")
 
-  let d = c.replace(/(^[\s\u200b]*|[\s\u200b]*$)/g, 'invisiblespace')
+  let d = c.replace(/(^[\s\u200b]*|[\s\u200b]*$)/g, "invisiblespace")
 
-  let e = d.split('invisiblespace')
+  let e = d.split("invisiblespace")
 
   let f = e.filter(Boolean)
 
   let g = f.join("")
 
-  let h = g.replace(/\n+/g, '\n')
-  
+  let h = g.replace(/\n+/g, "\n")
+
   return h
-  
-  }
+}
 
-const initialState = { ideBody: '# code here\nprint("Welcome to Treetop Learning")', termBuff: "", user: null, update: null }
+const initialState = {
```

Comment:
Codacy found an issue: [Unexpected line break after this opening brace.](https://app.codacy.com/gh/treetoplearning/heartwood/pullRequest?prid=5941503)